radars/primary/quadrants/open-edx/blips/ux-pattern-library.json

```diff
@@ -3,5 +3,5 @@
   "ring": "Hold",
   "quadrant": "Open edX",
   "isNew": "",
-  "description": "The <a href=\"https://github.com/edx/ux-pattern-library\">UX Pattern Library</a> is a prior iteration of a UI library and frontend styleguide for Open edX applications and sites.  Its use has been deprecated in favor of the Blip: Paragon design system.  It has not seen any investment since approximately 2016 and should be removed as a dependency where possible."
+  "description": "The <a href=\"https://github.com/openedx/ux-pattern-library\">UX Pattern Library</a> is a prior iteration of a UI library and frontend styleguide for Open edX applications and sites.  Its use has been deprecated in favor of the Blip: Paragon design system.  It has not seen any investment since approximately 2016 and should be removed as a dependency where possible."
```

Comment:
@davidjoy this repo didn't get moved to the openedx org. if it is only in edX's org, should it be on the tech radar? especially in the primary ring for open edx?

Reply:
Goint to turn this into an issue for when you come back :)